content/ja/foundations/colors.mdx

```diff
@@ -7,13 +7,13 @@ import { CalloutCardIdentity, ColorSet, ColorInfo, TopLink } from 'components';
 
 # カラー
 
-色はブランド認知の為の重要な側面の1つです。正しい色相 (色) を使用する為に確認を行うことが不可欠です。
+色はブランド認知の為の重要な側面の 1 つです。正しい色相 (色) を使用する為に確認を行うことが不可欠です。
```

Comment:
This was automatically formatted via Prettier. Is this okay?
It looks like it adjusted spacing.